mobile/Screens/SignUpScreen.jsx

```diff
@@ -101,20 +103,37 @@ export default function SignUp({ navigation }) {
     } else if (password === '') {
       Alert.alert('Please enter a password to proceed');
     } else if (password.length < 6) {
-      Alert.alert('Password must be longer than five');
+      Alert.alert('Password must be longer than five characters');
     } else {
       return true;
     }
     return false;
   };
 
-  const handleSignUp = () => {
+  const handleSignUp = async () => {
     if (!checkInputs()) {
       return;
     }
     setFirstName('');
     setEmail('');
     onChangePassword('');
+    try {
+      const userData = {
+        email,
+        password,
+      };
+      const res = await axios.post(`${process.env.EXPO_PUBLIC_SERVER_URL}/user/post`, userData);
+      if (res.data.error) {
+        console.error(res.data.error);
+      } else {
+        const userId = res.data._id;
+        Storage({ key: 'hello', value: userId, saveKey: true });
```

Comment:
Adding an `await` keyword before calling `Storage` ensures that the navigation to Home only occurs after the user ID has been set/stored